packages/@aws-cdk/aws-kinesis/test/integ.stream-dashboard.expected.json

```diff
@@ -4,6 +4,9 @@
       "Type": "AWS::Kinesis::Stream",
       "Properties": {
         "ShardCount": 1,
+        "StreamModeDetails": {
+          "StreamMode": "PROVISIONED"
```

Comment:
Check for ${StreamMode.PROVISIONED} for the exact enum values and same for ON_DEMAND occurences

Reply:
I've made this change in the unit tests. It's not applicable to these JSON files.